UE4_Online_Notes.md

```diff
@@ -39,7 +39,7 @@ The server can be run by one player. I does not need to be in a different machin
 > Official [doc](https://docs.unrealengine.com/en-US/Gameplay/Networking/Server/index.html).
 
 # Authority & Replication
-To detect, distinguish or decide code to be run in the server and/or in the clients, the function `bool` [AAcator::HasAuthority()](https://docs.unrealengine.com/en-US/API/Runtime/Engine/GameFramework/AActor/HasAuthority/index.html), returns `true` if it's on the server and `false` otherwise.
+To detect, distinguish or decide code to be run in the server and/or in the clients, the function `bool` [AActor::HasAuthority()](https://docs.unrealengine.com/en-US/API/Runtime/Engine/GameFramework/AActor/HasAuthority/index.html), returns `true` if it's on the server and `false` otherwise.
```

Comment:
How would a review look like? Let's push the button and see.

Reply:
```suggestion
To detect, distinguish or decide code to be run in the server and/or in the clients, the function `bool` [AActor::HasAuthority()](https://docs.unrealengine.com/en-US/API/Runtime/Engine/GameFramework/AActor/HasAuthority/index.html), returns `true` if this code is being run on the server and `false` otherwise.
```